recipe/meta.yaml

```diff
@@ -18,9 +18,17 @@ build:
     # The "blas" package must always track a feature, else it will be
     # prioritised over all others.
     - blas_{{ variant }}
+  {% if variant == "openblas" %}
+  script:                                             # [unix]
+    - bash {{ RECIPE_DIR }}/symlink_{{ variant }}.sh  # [unix]
+  {% endif %}
```

Comment:
As noted elsewhere, I'd rather this live in the packages implementing BLASes and LAPACKs. This makes it easier for us to fix if things change later for any one of these packages. Also it is easier to reason about what is happening.